### PR TITLE
[example/bwmf] Added bwmf matrix datautil for the new representation. 

### PR DESCRIPTION
**Pull request by BaiGang**
Also did go fmt and git ignored the generated excutables.

### CODE REVIEW

File: example/bwmf/datautil/main.go [hunk 1]
```diff
@@ -97,48 +99,58 @@ func txtToMatPb() (*pb.MatrixShard, error) {
...
 }
 
 func saveResult(shard *pb.MatrixShard, config *bwmf.Config, path string) error {
-	client, err := GetFsClient(config)
+	client, err := bwmf.GetFsClient(config)
 	if err != nil {
 		return fmt.Errorf("Failed getting filesystem.Client: %s", err)
 	}
+
+	fmt.Println(shard)
```
Comment (5kg) [inline example/bwmf/datautil/main.go]: Remove this?

Comment (BaiGang) [inline example/bwmf/datautil/main.go]: Yes. Thanks!

File: example/bwmf/datautil/main.go [hunk 2]
```diff
@@ -72,24 +73,25 @@ func loadConfig(path string) (*bwmf.Config, error) {
 	return config, nil
 }
 
-func txtToMatPb() (*pb.MatrixShard, error) {
-	// Each line represents an elem, formated as "rowID columnID count".
-	// The `rowID`s are guaranteed to be sorted and grouped (as in reducer tasks of a Hadoop job).
+// NOTE(baigang): Each line represents an elem, formated as "rowID columnID count".
```
Comment (5kg) [inline example/bwmf/datautil/main.go]: s/elem/element/
s/count/value/

**Comment by 5kg** [comment]
LGTM

File: example/bwmf/datautil/main.go (hunk 2 above)
Comment (BaiGang) [inline example/bwmf/datautil/main.go]: OK. Let's fix this in next push.

File: example/bwmf/datautil/main.go [hunk 3]
```diff
@@ -97,40 +99,47 @@ func txtToMatPb() (*pb.MatrixShard, error) {
...
-		shard.Row = append(shard.Row, rowData)
+		ir = append(ir, uint32(len(val)))
 	}
-	log.Println("num of rows: ", len(shard.Row))
-	return shard, nil
+	log.Println("num of rows: ", len(ir)-1)
+	log.Println("num of elems: ", len(val))
+
+	return &pb.MatrixShard{
+		IsSparse: false,
```
Comment (5kg) [inline example/bwmf/datautil/main.go]: I missed this oops.

s/false/true/

Comment (BaiGang) [inline example/bwmf/datautil/main.go]: Ok. Let's do a quick fix.

**Pull request merged**